main.py

```diff
@@ -34,32 +35,21 @@ def setup(self):
         self.traces = {}
         self.last_id = 30000
         self.automate = Automate(self)
+        self.scheduler = Scheduler()
         if settings.TRIGGER_SCHEDULE_TRACES:
-            event = KytosEvent('amlight/scheduler.add_job')
-            event.content['id'] = 'automatic_traces'
-            event.content['func'] = self.automate.run_traces
-            try:
-                trigger = settings.SCHEDULE_TRIGGER
-                kwargs = settings.SCHEDULE_ARGS
-            except AttributeError:
-                trigger = 'interval'
-                kwargs = {'seconds': 60}
-            event.content['kwargs'] = {'trigger': trigger}
-            event.content['kwargs'].update(kwargs)
-            self.controller.buffers.app.put(event)
+            id_ = self.automate.schedule_id('automatic_traces')
```

Comment:
@Alopalao let's encapsulate all of this on `schedule_traces(settings=settings)` including the if conditional with an early return, and then `schedule_traces` only return the job that was scheduled, it's almost there, just a matter of moving the statements. Let's also pass `settings` as a arg just so it becomes slightly easier when you pass a Mock object.  Same idea for `automate.schedule_important_traces(settings=settings)`  with the conditional below.

Reply:
Does this mean that `Scheduler()` is going to be called inside Automate?